akka-stream/src/main/scala/akka/stream/impl/Consts.scala

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.impl
+
+import akka.japi.function.{ Function ⇒ JFun, Function2 ⇒ JFun2 }
+import akka.japi.{ Pair ⇒ JPair }
+
+private[stream] object Consts {
+  private[this] val JavaIdentityFunction = new JFun[Any, Any] {
+    @throws(classOf[Exception]) override def apply(param: Any): Any = param
+  }
+
+  val JavaPairFunction = new JFun2[AnyRef, AnyRef, AnyRef JPair AnyRef] {
+    def apply(p1: AnyRef, p2: AnyRef): AnyRef JPair AnyRef = JPair(p1, p2)
+  }
```

Comment:
maybe add a TODO that we should provide those cached functions in `akka.japi.Util` directly